packages/import-mistica-icons/index.js

```diff
@@ -194,6 +198,35 @@ const createIconComponentSource = async (name, componentName, svgIconsInfo) => {
     return format(source);
 };
 
+/**
+ * @typedef {{[key: string]: Array<string>}} iconKeywordsInfo
```

Comment:
use PascalCase for type names (`IconKeywordsInfo`)

Reply:
good catch. fixed